src/cascadia/TerminalApp/GlobalAppSettings.cpp

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+#include <WinUser.h>

```

Comment:
This should probably go into the pch.h if it isn't there already.

Reply:
Seems that we do not need this header. removed.